### PR TITLE
added Company

### PR DESCRIPTION
**Pull request by elitekali**
Travel Company in Kashmir

### CODE REVIEW

**Comment by elitekali** [comment]
added company

**Comment by elitekali** [comment]
> 

yes

**Comment by abhishekagarwal87** [comment]
you need to add details about how are you using Apache Druid.

**Comment by elitekali** [comment]
I Have already Updated the Details

File: druid-powered.md [hunk 1]
```diff
@@ -822,6 +822,20 @@ Learn more about TrafficGuard’s comprehensive fraud mitigation at (https://www
...
 
+## Travel Clan
+
+[Travel Clan](https://www.travelclan.in/) is a premier travel agency located in the scenic city of Srinagar, Kashmir. We offer a comprehensive range of [Kashmir tour packages](https://www.travelclan.in/kashmir-tour-packages/) that cater to the needs and preferences of all kinds of travellers, whether you're looking for adventure, relaxation, or culture. Our packages include everything from transportation and accommodation to sightseeing tours and adventure activities, ensuring that you have a hassle-free and enjoyable vacation.
+
+We also provide [Kashmir Tour Package from Kolkata](https://www.travelclan.in/tour/kashmir-tour-package-from-kolkata/) which includes most of the tourist [Places to visit in Kashmir](https://www.travelclan.in/places-to-visit-in-kashmir/)
+
+Travel Clan uses Druid to ingest real-time data from various sources such as hotel booking systems, travel booking portals, and social media platforms to get insights into real-time trends and preferences of our customers. Druid helps us track the operational metrics such as employee productivity, inventory management, and resource utilization to optimize our operations and reduce costs.
```
Comment (abhishekagarwal87) [inline druid-powered.md:831]:
```suggestion
[Travel Clan](https://www.travelclan.in/) uses Druid to ingest real-time data from various sources such as hotel booking systems, travel booking portals, and social media platforms to get insights into real-time trends and preferences of our customers. Druid helps us track the operational metrics such as employee productivity, inventory management, and resource utilization to optimize our operations and reduce costs.
```

**Comment by jamsheed861** [comment]
add this company [Kashmir cab service](https://www.travelmykashmir.com). We provide all tour and travel services including [bike rental in srinagar](https://www.bikerentalsrinagar.com) , [Kashmir tour packages](https://travelmykashmir.com/kashmir-tour-packages-2/
), [Taxi services in Srinagar](https://travelmykashmir.com/kashmir-car-rental/
)

**Pull request closed without merging**